apis/v1alpha1/resource_mapping.pb.go

```diff
@@ -126,6 +126,16 @@ type Resource struct {
 	// Required. The full resource name. Example:
 	// `//spanner.googleapis.com/projects/p1/instances/i1/databases/d1`
 	Name string `protobuf:"bytes,2,opt,name=name,proto3" json:"name,omitempty"`
+	// Optional. The representation for subresource in the format of a URL.
+	// Subresource is the resource thats beyond the cloud resource level,
+	// like spanner table, GCS object.
+	// Format: "hierarchy_string?qualifier_string"
+	// where hierarchy_string represents the hierarchy of the subresource to this parent
+	// in the format of parent/foo/child/bar.
+	// and qualifier_string represents the additional information of the subresource
+	// in the format of: key1=value1&key2=value2.
+	// Example: rows with provider=gcp in a database table1 would be: `table/table1?provider=gcp`
```

Comment:
use another example, `provider=gcp` is confusing here as we have `provider` as a field of the `resource_mapping`

Reply:
updated to userID=abcxyz